src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx

```diff
@@ -32,14 +32,22 @@ Adds a user-defined attribute name and value to subsequent events on the page.
 
 Agent version [nr-593](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) or higher.
 
+For `persist` parameter or `null` value support, agent version 1.230.0 or higher is required.
+
 ## Description
 
-Adds a user-defined attribute name and value to subsequent events on the page. Make this call before the window load event fires (when that data is transmitted) in order for the attributes to be included in the [`PageView`](/docs/insights/explore-data/attributes/browser-default-attributes-insights#browser-attributes-table) event. Once an attribute is set, the New Relic platform records it with all [`PageAction`](/docs/insights/explore-data/attributes/browser-default-attributes-insights#pageaction-list) events until the page is reloaded or the attribute is manually unset.
+Make this call before the window load event fires (when that data is transmitted) in order for the attributes to be included in the [`PageView`](/docs/insights/explore-data/attributes/browser-default-attributes-insights#browser-attributes-table) event. Once an attribute is set, the New Relic platform records it with all events until the page is reloaded or the attribute is manually unset.
 
-If you are using [SPA monitoring](/docs/browser/single-page-app-monitoring/get-started/welcome-single-page-app-monitoring) with a compatible agent version, attributes set with this call will also be included in [`newrelic.interaction` events](/docs/browser/new-relic-browser/browser-agent-apis/browser-spa-api-newrelicinteraction). However, attributes set using the SPA API will take precedence over these attributes.
+If you are using [SPA monitoring](/docs/browser/single-page-app-monitoring/get-started/welcome-single-page-app-monitoring) with a compatible agent version, attributes set with this call will also be included in [`newrelic.interaction`](/docs/browser/new-relic-browser/browser-agent-apis/browser-spa-api-newrelicinteraction) events. However, attributes set using the SPA API will take precedence over these attributes.
 
 Errors for custom attributes will be included in events on the [JS Errors page](/docs/browser/new-relic-browser/browser-pro-features/javascript-errors-page-detect-analyze-errors). To view or log errors for a custom attribute via API, use the browser API's [`noticeError`](/docs/browser/new-relic-browser/browser-agent-spa-api/newrelicnoticeerror-browser-agent-api) call.
 
+With the `persist` flag, the attribute can also be in stored in the browser, so that subsequent page visits of the _same_ origin **within a session** span retain it on events. Do note that this functionlity may fluctuate depending on end-user browser privacy settings. If this function is called with a `value = null`, the attribute will be deleted from **both** the current page's events and the storage, _regardless_ of the `persist` flag.
```

Comment:
```suggestion
With the `persist` flag, the attribute can also be in stored in the browser, so that subsequent page visits of the _same_ origin **within a session** span retain it on events. Do note that this functionality may fluctuate depending on end-user browser privacy settings. If this function is called with a `value = null`, the attribute will be deleted from **both** the current page's events and the storage, _regardless_ of the `persist` flag.
```